tests/helpers/has-attributes.js

```diff
@@ -0,0 +1,24 @@
+import QUnit from 'qunit';
+
+/**
+  Creates a custom QUnit Assertion.
+  This assertion will compare 2 arrays of attributes.
+
+  @method hasAttributes
+  @param {Array} attributes
+  @param {Array} attributesToTest
+*/
+function compareArrays(attributes, attributesToTest) {
+  attributesToTest.sort();
```

Comment:
I think `attributesToTest` is misleading when we're testing both. I would say the first param should be `actual` and the second `expected`.